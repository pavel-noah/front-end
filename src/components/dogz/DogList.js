import React from 'react';
import PropTypes from 'prop-types';
import Dog from './Dog';

export default function DogList({ dogs }) {
    const dogElement = dogs.map(dog => {
        return (
            <li key={dog._id} style={{ listStyle: 'none' }}>
                <Dog dogName={dog.name} dogAge={dog.age} dogWeight={dog.weight} />
            </li>
        );
    });

    return (<>
            <ul>
                {dogElement}
            </ul>
        </>
    );
}

DogList.propTypes = {
    dogs: PropTypes.array.isRequired
};
