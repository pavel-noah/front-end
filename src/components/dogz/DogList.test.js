import React from 'react';
import { shallow } from 'enzyme';
import DogList from './DogList';

describe('dog list', () => {
    it('renders dog list', () => {
        const wrapper = shallow(<DogList dogs={[]} />);
        expect(wrapper).toMatchSnapshot();
    });
});
  
