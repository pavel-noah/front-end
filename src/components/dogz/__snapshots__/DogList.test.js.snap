// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dog list renders dog list 1`] = `
<Fragment>
  <ul />
</Fragment>
`;
